.changeset/unlucky-beans-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': patch
+---
+
+`ERC2771Context`: Return the forwarder address whenever the `msg.data` of a call originating from a trusted forwarder is not long enough to contain the request signer address (i.e. `msg.data.length` is less than 20 bytes).
```

Comment:
```suggestion
`ERC2771Context`: Return the forwarder address whenever the `msg.data` of a call originating from a trusted forwarder is not long enough to contain the request signer address (i.e. `msg.data.length` is less than 20 bytes), as specified by ERC-2771.
```

Reply:
Thoughts on?

```suggestion
`ERC2771Context`: Return the forwarder address whenever the `msg.data` of a call originated from a trusted forwarder is not long enough to contain the request signer address (i.e. `msg.data.length` is less than 20 bytes), as specified by ERC-2771
```

Reply:
Either is good for me. When you reach concensus you can merge without my formal approval.

Reply:
No issues 👌